resend.yaml

```diff
@@ -2127,499 +3215,580 @@ components:
+      type: object
+      properties:
+        object:
+          type: string
+          description: Type of the response object.
+          example: list
+        has_more:
+          type: boolean
+          description: Indicates if there are more segments to retrieve.
+          example: false
+        data:
+          type: array
+          description: Array containing segment information.
+          items:
+            type: object
+            properties:
+              id:
+                type: string
+                description: The ID of the segment.
+                example: '78261eea-8f8b-4381-83c6-79fa7120f1cf'
+              name:
+                type: string
+                description: The name of the segment.
+                example: 'Registered Users'
+              created_at:
+                type: string
+                format: date-time
+                description: Timestamp indicating when the segment was created.
+                example: '2023-10-06T22:59:55.977Z'
+    ListContactTopicsResponseSuccess:
+      type: object
+      properties:
+        object:
+          type: string
+          description: Type of the response object.
+          example: list
+        has_more:
+          type: boolean
+          description: Indicates if there are more topics to retrieve.
+          example: false
+        data:
+          type: array
+          description: Array containing topic subscription information.
+          items:
+            type: object
+            properties:
+              id:
+                type: string
+                description: The ID of the topic.
+                example: 'b6d24b8e-af0b-4c3c-be0c-359bbd97381e'
+              name:
+                type: string
+                description: The name of the topic.
+                example: 'Weekly Newsletter'
+              subscription:
+                type: string
+                enum:
+                  - opt_in
+                  - opt_out
+                description: The subscription status.
+                example: 'opt_in'
+              created_at:
+                type: string
+                format: date-time
+                description: Timestamp indicating when the topic was created.
+                example: '2023-04-08T00:11:13.110779+00:00'
+    UpdateContactTopicsOptions:
+      type: object
+      required:
+        - topics
+      properties:
+        topics:
+          type: array
+          description: Array of topic subscription updates.
+          items:
+            type: object
+            required:
+              - id
+              - subscription
+            properties:
+              id:
+                type: string
+                description: The Topic ID.
+              subscription:
+                type: string
+                enum:
+                  - opt_in
+                  - opt_out
+                description: The subscription action.
+    UpdateContactTopicsResponseSuccess:
```

Comment:
This also feels weird. The API is only returning an ID?

https://github.com/resend/resend-api/blob/1467ffa9af287dd254dcfd8d9b5f9c60cc4d710c/src/routes/contacts/topics/patch.ts#L86

But it seems like this says it returns the `object`, and a `data` array?